src/libs/date-utils.ts

```diff
@@ -26,4 +26,20 @@ export const parseDate = (date?: string): Date | undefined => {
   const datePart = dateString.substring(0, offsetIndex);
   const zonePart = dateString.substring(offsetIndex + 1).replace("GMT", "UTC");
   return DateTime.fromFormat(datePart, format).setZone(zonePart);
-};
\ No newline at end of file
+};
+
+/**
+ * 
+ * Calculates the difference between last crawl time and now
+ * 
+ * @param lastCrawl date of last crawl
+ * @returns difference in minutes
+ */
+ export const calculateMinutesPassed = (lastCrawl: string ) => {
+  const lastCrawlDate = parseDate(lastCrawl);
+  const now = new Date();
+
+  const difference = now.getTime() - lastCrawlDate?.getTime();
+  // TODO: +180 accounting for GMT+3, will need improving
+  return Math.round(difference / 60000) + 180;
```

Comment:
This project uses luxon which supports timezone handling by default. Refactor to use it

Reply:
I've confused myself thinking about this.
If the elastic and lambda are hosted in the same location no time zone adjustment is needed?
If they are not in the same place I need to find out where they are and adjust accordingly...